pkg/api/applications/v2/activity.go

```diff
@@ -107,6 +107,13 @@ type RefreshActivity struct {
 	ActivityFailure
 }
 
+type ActivityPatchRequest struct {
+	Title string   `json:"title"`
+	Tags  []string `json:"tags"`
+	// Data is a JSON-serializable value for internal metadata about the Activity
+	Data any `json:"_stormforge,omitempty"`
```

Comment:
Any particular reason for `any` versus `ActivityFailure`?

Reply:
If you wanted to pass another structure in the future, you'd be able to without modifying the code or creating a new variant. If `ActivityFailure` is the only type of struct we ever want to pass as the metadata, then I'd be fine with that. Seems like it defeats the purpose of the endpoint accepting arbitrary JSON.